### ISSUE TITLE
Custom Fusion Multiblocks via KubeJS cannot process recipes above IV

### ISSUE THREAD

**Issue by Xefyr0** [GregTechCEu/GregTechCEu-1.19#1905]
### Checked for existing issues

- [X] I have checked for existing issues, and have found none.

### Tested latest version

- [X] I have checked that this occurs on the latest version.

### GregTech CEu Version

1.4.0

### Minecraft Version

1.20.1

### Recipe Viewer Installed

EMI

### Environment

Singleplayer

### Cross-Mod Interaction

No

### Other Installed Mods

EMI, KubeJS, and their dependencies

### Expected Behavior

Creating a custom fusion reactor (for parallel recipes) using KubeJS would result in the reactor being to process all voltage tiers of recipes with the same starting energy requirement toroidal fusion reactors have

### Actual Behavior

KJS Fusion reactor processes IV recipes (such as Helium plasma) just fine, but higher voltage tier recipes (Like Tritanium and Neutronium) process for exactly one tick & stop for no reason.

### Steps to Reproduce

Using a 1.20.1 instance with GTCEU, KubeJS and their dependencies, create a new Creative world.
Put the `startup.js` file in the attached archive into `.minecraft/kubejs/startup_scripts`
[startup script archive](https://github.com/user-attachments/files/16918838/startup.zip)
Form the "helical fusion reactor" multiblock with the Terminal, and hook up power. 
Input the fluids for Tritanium (Titanium and Duranium) or Neutronium recipes (Naquadria and Americium) into hatches
Recipe processes for exactly one tick, with associated sounds.

**Comment by Xefyr0** [comment]
UPDATE: I have found the cause of the problem thanks to reverse engineering the working parallel fusion reactor the ATM devs made.

My multiblock was tiered at UHV, giving it an internal capacity of 160MEU. That was less than the starting energy requirement of Tritanium or Neutronium, the LuV and UV recipes that I tested.
EDIT: the internal buffer of the fusion reactor _does_ increase as more input energy hatches are added depending on tier. In most cases the maximum is two, though.

Compare this with the UIV-tier Mega Fusion Reactor in ATM9, which had an internal capacity of 640MEU, and thus could easily provide the 400MEU requirement for all GTCEU fusion recipes.
